extensions/amp-ad/0.1/test/test-concurrent-load.js

```diff
@@ -75,13 +76,27 @@ describes.realWin('concurrent-load', {}, env => {
       installTimerService(win);
     });
 
-    it('should throttle ad loading one per second', () => {
+    it('should throttle ad loading one per second', function* () {
       expect(is3pThrottled(win)).to.be.false;
       incrementLoadingAds(win);
       expect(is3pThrottled(win)).to.be.true;
       clock.tick(999);
+      yield macroTask();
```

Comment:
why do we want to treat the 2 ticks differently? isn't it cheating? are we doing a test to just get it passed?

the test is translated to:
1. clock tick 999ms
2. ad should still be throttled  in the next event loop (1ms to go!)
3. clock tick 1ms
4. ad should NOT be throttled starting from the next event loop